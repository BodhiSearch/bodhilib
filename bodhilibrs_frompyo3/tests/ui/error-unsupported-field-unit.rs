use frompyo3::FromPyO3;

#[derive(FromPyO3)]
struct TestUnsupportedUnit(i32);

fn main() {}
