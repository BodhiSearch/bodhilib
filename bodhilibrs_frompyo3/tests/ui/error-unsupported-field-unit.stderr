error: Unsupported field format, only named fields supported
 --> tests/ui/error-unsupported-field-unit.rs:4:27
  |
4 | struct TestUnsupportedUnit(i32);
  |                           ^^^^^
